bakery/discharge_test.go

```diff
@@ -28,7 +28,7 @@ func (*DischargeSuite) TestDischargeAllNoDischarges(c *gc.C) {
 	rootKey := []byte("root key")
 	m, err := macaroon.New(rootKey, []byte("id0"), "loc0", macaroonCurrentVersion)
 	c.Assert(err, gc.IsNil)
-	ms, err := bakery.DischargeAll(m, noDischarge(c))
+	ms, err := bakery.DischargeAll(context.TODO(), m, noDischarge(c))
```

Comment:
context.TODO is for occasions when a call needs a context and you know you
should be providing a context taken from the caller, but you don't have
that context yet. It's not actually right to use here - I should be using BC (background
context) instead.